GOVERNANCE.md

```diff
@@ -53,7 +53,7 @@ and invited to the [private maintainer mailing list](https://lists.cncf.io/g/cnc
 Time zones permitting, Maintainers are expected to participate in the public
 developer meeting, which occurs
 
-Every Monday at 9am EST ([video call link](https://meet.google.com/trx-vqbw-ekb))
+Every two weeks, on Monday, at 9am EST ([video call link](https://meet.google.com/trx-vqbw-ekb)).
```

Comment:
Actually, it might be better to replace the meeting link with a link to the google group to follow what @elsony said: https://kubernetes.slack.com/archives/C02SX9E5B55/p1701461002273219

```suggestion
Every two weeks, on Monday, at 9am EST. Please join the [Google group](https://groups.google.com/g/devfiles-community) to get an invite.
```